source/learners/sfd_discussions/find_follow_conversations.rst

```diff
@@ -0,0 +1,155 @@
+.. _find-follow-conversations:
+
+##############################################
+Finding and Following the Right Conversations
+##############################################
+
+Discussions can grow quickly as a course progresses. 
+Knowing how to browse and search helps you save time and discover conversations that matter to you. 
+
+.. contents::
+  :local:
+  :depth: 1
+
+.. _Navigating the Discussions page:
+
+********************************
+Navigating the Discussions page
+********************************
+
+When you open the Discussions tab in your course, you see four tabs at the top, as shown in the image below. 
+
+1. **My Posts:** Shows all posts that you’ve written or taken part in through a response or comment. Click a post title in the left panel to open it and see its responses and comments.
+
+2. **All Posts:** Lists every post in the course discussions. By default, posts with the most recent activity appear at the top.
+
+3. **Topics:** Displays all discussion topics in the course. Topics follow the same three-level hierarchy as the course content:
+   
+   a. The top level represents course sections.  
+   b. The second level shows subsections.  
+   c. The third level lists units, which are the actual discussion topics that contain posts. Select a subsection to expand it and see its topics. When you click a topic name, you will see the posts related to it.
+
+4. **Learners:** Shows everyone who has written a post, response, or comment. Click a username to see their contributions. Under each name you can see the number of posts they created and their total contributions.
+
+.. tip:: Use *My Posts* for tracking your own activity and *Topics* when you want to focus on the part of the course you are studying.
+
+
+.. figure:: /_images/learners/Discussions_page.png
+   :alt: Screenshot of the discussion page
```

Comment:
This is not a helpful caption for people with screenreaders. A caption should be descriptive of what someone who cannot see is missing. For example, "The discussion page, highlighting the top left Tabs, top right search bar, and top right "Add a Post" button"